test/e2e/lib/gutenberg/blocks/slideshow-block-component.js

```diff
@@ -12,6 +12,21 @@ class SlideshowBlockComponent extends GutenbergBlockComponent {
 	static blockTitle = 'Slideshow';
 	static blockName = 'jetpack/slideshow';
 	static blockFrontendSelector = By.css( '.entry-content .wp-block-jetpack-slideshow' );
+
+	/**
+	 * Uploads images to the slideshow.
+	 *
+	 * @param {{imageName: string, fileName: string, file: string}} filesDetails a list of fileDetails
```

Comment:
I'd follow [this guide](https://jsdoc.app/tags-param.html#parameters-with-properties) on this one:
```js
 * @param {string} filesDetails.imageName
 * @param {string} filesDetails.fileName
 * @param {string} filesDetails.file
```

Reply:
👍🏻

Reply:
Actually, didn't notice that it's an array of objects. In this case this should be:
```suggestion
	 * @param {{imageName: string, fileName: string, file: string}[]} filesDetails a list of fileDetails
```